### ISSUE TITLE
FEAT: add `explain` option after running a query

### ISSUE THREAD

**Issue by Zerotask** [beekeeper-studio/beekeeper-studio#784]
**Is your feature request related to a problem? Please describe.**
No direct problem, but in some cases an EXPLAIN / DESCRIBE can be informative.

**Describe the solution you'd like**
A simple one-click option to run a EXPLAIN / DESCRIBE for a given query.

**Describe alternatives you've considered**
write it by yourself - although there can be differences among the database systems. The GUI could always provide the specific EXPLAIN method.

**Additional context**
Example from the tool *Adminer*
![image](https://user-images.githubusercontent.com/20150243/131365358-5ad9d2c0-d9c1-4ecc-970f-69326ad8601c.png)

Beekeeper could add this similarily in the footer, next to the execution time:
![image](https://user-images.githubusercontent.com/20150243/131365481-58c786e4-0201-4a85-ad8f-080193fec25a.png)

**Comment by rathboma** [comment]
Yeah I think a special `explain` UI would be helpful. Again not a priority, but definitely something we want eventually!
